docs/source/markdown/options/add-host.md

```diff
@@ -1,6 +1,6 @@
 ####> This option file is used in:
 ####>   podman build, create, pod create, run
-####> If you edit this file, make sure your changes
+####> If file is edited, make sure the changes
```

Comment:
```suggestion
####> If this file is edited, make sure the changes
```

or perhaps `the`

Reply:
Oh please no. If it's absolutely necessary to un-you this part, *please* make it a separate PR. This PR is already unwieldy enough.